PostProcessing/Runtime/PostProcessRenderContext.cs

```diff
@@ -23,28 +23,42 @@ public Camera camera
             {
                 m_Camera = value;
 
-                if (XRSettings.isDeviceActive)
+                if (m_Camera.stereoEnabled)
                 {
 #if UNITY_2017_2_OR_NEWER
-                    RenderTextureDescriptor xrDesc = XRSettings.eyeTextureDesc;
+                    var xrDesc = XRSettings.eyeTextureDesc;
                     width = xrDesc.width;
                     height = xrDesc.height;
+                    m_sourceDescriptor = xrDesc;
 #else
-                    width = XRSettings.eyeTextureWidth; // double this for single-pass when we can't query eyeTextureDesc
+                    // Single-pass is only supported with 2017.2+ because
+                    // that is when XRSettings.eyeTextureDesc is available.
+                    // Without it, we don't have a robust method of determining
+                    // if we are in single-pass.  Users can just double the width
+                    // here if they KNOW they are using single-pass.
+                    width = XRSettings.eyeTextureWidth;
                     height = XRSettings.eyeTextureHeight;
 #endif
 
-                    if (camera.stereoActiveEye == Camera.MonoOrStereoscopicEye.Right)
+                    if (m_Camera.stereoActiveEye == Camera.MonoOrStereoscopicEye.Right)
                         xrActiveEye = (int)Camera.StereoscopicEye.Right;
 
-                    xrSingleEyeWidth = XRSettings.eyeTextureWidth;
```

Comment:
RuntimeUtilities.cs L449 needs to be updated for this

Reply:
Nice, thanks.